assets/src/edit-story/components/canvas/layout.js

```diff
@@ -114,7 +107,50 @@ const Area = styled.div`
 
 // Page area is not `overflow:hidden` by default to allow different clipping
 // mechanisms.
-const PageArea = styled(Area).attrs({ area: 'page', overflowAllowed: true })``;
+const PageAreaContainer = styled(Area).attrs({
+  area: 'page',
+  overflowAllowed: false,
+})`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background-color: ${({ theme }) => theme.colors.fg.v1};
+`;
+
+const PageAreaContent = styled.div`
+  width: 100%;
+  height: var(--page-height-px);
+  overflow: visible;
+  position: relative;
+`;
+
+const PageAreaSafeZone = styled.div`
+  pointer-events: none;
+  position: absolute;
+  background-image: linear-gradient(
+    45deg,
+    #a3a3a3 25%,
+    #000000 25%,
+    #000000 50%,
+    #a3a3a3 50%,
+    #a3a3a3 75%,
+    #000000 75%,
+    #000000 100%
+  );
```

Comment:
FYI: This can be done more compactly using a [repeating linear gradient with multi-position stops](https://developer.mozilla.org/en-US/docs/Web/CSS/repeating-linear-gradient#Syntax) as:

```
  background-image: repeating-linear-gradient(45deg, #a3a3a3 25%, #000000 0% 50%);
```